card.go

```diff
@@ -24,8 +24,8 @@ type CardFunding string
 // For more details see https://stripe.com/docs/api#create_card and https://stripe.com/docs/api#update_card.
 type CardParams struct {
 	Params
-	Token                                         string
-	Customer, Recipient                           string
+	External_Account, Token                       string
```

Comment:
> How would you handle this? From that link it tells me it's either a token or a dictionary (similar to the current implementation for customer).

Good question. I notice that the bindings seem to have just not bothered for support on the dictionary version for customer, so it seems fine to leave it out of here for now too.